lib/Analysis/GateAnalysis.cc

```diff
@@ -339,12 +308,12 @@ bool GateAnalysisPass::runOnModule(llvm::Module &M) {
     if (!F.isDeclaration())
       changed |= runOnFunction(F, CDP.getControlDependenceAnalysis(F));
 
-  if (GsaReplacePhis)
-    for (auto &F : M)
-      if (!F.isDeclaration())
-        for (auto &BB : F)
-          for (auto &I : BB)
-            assert(!isa<PHINode>(I));
+  LOG("gsa", if (GsaReplacePhis) for (
```

Comment:
ouch on formatting. You can fix it by moving `if` to next line, maybe adding extra blank line before it. Then `clang-format` will not move it back up.